packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +188,12 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.leading], the intended use for the return value of this method.
   Widget? buildLeading(BuildContext context);
 
+  /// [AppBar.automaticallyImplyLeading]
+  bool? automaticallyImplyLeading;
+
+  /// [AppBar.leadingWidth]
```

Comment:
Here we should use `/// {@macro flutter.material.appbar.leadingWidth}` . 

And in `app_bar.dart`:
https://github.com/flutter/flutter/blob/7f20e5d18ce4cb80c621533090a7c5113f5bdc52/packages/flutter/lib/src/material/app_bar.dart#L644-L649

change:
`leading` -> `AppBar.leading`, and `leadingWidth` -> `AppBar.leadingWidth`.